elasticsearch-extensions/lib/elasticsearch/extensions/test/cluster.rb

```diff
@@ -299,23 +296,25 @@ def start
           def stop
             begin
               nodes = __get_nodes
-            rescue Exception => e
-              __log "[!] Exception raised when stopping the cluster: #{e.inspect}".ansi(:red)
+            rescue StandardError => e
+              __log "[!] Error raised when stopping the cluster: #{e.inspect}".ansi(:red)
               nil
             end
 
-            return false if nodes.nil? or nodes.empty?
+            return false if nodes.nil? || nodes.empty?
 
-            pids  = nodes['nodes'].map { |id, info| info['process']['id'] }
+            pids = nodes['nodes'].map { |id, info| info['process']['id'] }
 
-            unless pids.empty?
-              __log "Stopping Elasticsearch nodes... ".ansi(:faint), :print
+            if pids.empty?
+              return false
```

Comment:
Nesting, style: If you're `return`ing, you don't need the additional else and its nesting.